x-pack/plugins/observability/server/services/slo/sli_client.ts

```diff
@@ -68,7 +64,11 @@ export class DefaultSLIClient implements SLIClient {
         continue;
       }
 
-      indicatorDataBySlo[slo.id] = handleResult(dateRangeBySlo[slo.id], result.responses[i]);
+      // @ts-ignore
```

Comment:
What is TS complaining about here?

Reply:
result.responses[i] has an `aggregations` property, but I cannot figure out how to type it correctly.